churn-risk/src/app.py

```diff
@@ -1,184 +1,193 @@
-from test.e2e import walkthrough
-
 import pandas as pd
-from h2o_wave import app, main, Q, ui
-from plotly import graph_objects as go
+import os
 
+from pygments import highlight
+from pygments.formatters.html import HtmlFormatter
+from pygments.lexers import get_lexer_by_name
+from h2o_wave import app, main, Q, ui, data
 from .churn_predictor import ChurnPredictor
-from .config import Configuration
-from .plots import (
-    convert_plot_to_html,
-    generate_figure_pie_of_target_percent,
-    get_image_from_matplotlib,
-    tall_stat_card_dollars,
-    wide_stat_card_dollars,
-)
-from .utils import python_code_content
-
-config = Configuration()
+
+df = pd.read_csv('data/churnTest.csv')
+df.dropna(subset=['State', 'Account_Length', 'Area_Code', 'Phone_No'], inplace=True)
+df.fillna({'International_Plan': 'no', 'Voice_Mail_Plan': 'no', 'No_Vmail_Messages': 0, 'Total_Day_minutes': 0.00,
```

Comment:
The data actually being used to train and get predictions in our predictive model has missing values. I think it is better to drop these rows from plotting completely rather than replace with default values in the visualizations as the predictions we are showing were from when the values with None, not the default value

Reply:
If I understand correctly, this should do the trick:
```py
df.dropna(inplace=True)
```
as we want to drop N/A values for all the cols, right?